pom.xml

```diff
@@ -69,13 +69,6 @@
         <tag>HEAD</tag>
     </scm>
 
-    <distributionManagement>
```

Comment:
The` central-publishing-maven-plugin` 0.7.0+ supports snapshot releases directly. Publishing with a -SNAPSHOT version will automatically upload the component to the right location.